test/connection/cross_site_forgery_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+require 'stubs/test_server'
+
+class ActionCable::Connection::CrossSiteForgeryTest < ActiveSupport::TestCase
```

Comment:
Missing a test for these two cases:

1. origin is not present.
2. origin is an invalid URL.